### PR TITLE
fix(sdk): open Github issues on Windows/WSL in case of Rust panic, letting the process terminate

### LINKED ISSUE #11373: `Would you like to create a Github issue? » Yes` does nothing in terminal on Windows
```
C:\Users\Jan\Documents\throwaway\11318>npx prisma migrate dev
Environment variables loaded from .env
Prisma schema loaded from prisma\schema.prisma
Datasource "db": MySQL database "amethyst_skink" at "mysql-db-provision.cm0mkpwj8arx.eu-central-1.rds.amazonaws.com:3306"

Oops, an unexpected error occured!
Error in migration engine.
Reason: [libs\sql-schema-describer\src\walkers.rs:592:14] ...

Please create an issue with your `schema.prisma` at
https://github.com/prisma/prisma/issues/new


Please help us improve Prisma by submitting an error report.
Error reports never contain personal or other sensitive information.
Learn more: https://pris.ly/d/telemetry

√ Submit error report » No
√ Would you like to create a Github issue? » Yes
```

3.8.1

### PR DESCRIPTION
**Pull request by jkomyno**
Closes https://github.com/prisma/prisma/issues/11373.
Deprecates https://github.com/prisma/prisma/pull/12678 and https://github.com/prisma/prisma/pull/12692.

This PR introduces the following dependencies:
- `is-windows`
- `is-wsl`

### Setup

- `mkdir reproduction-13356 && cd reproduction-13356`
- `npx prisma init --datasource-provider sqlite`

### On Windows

1. The following block should open a new Github issue without terminating the `prisma` process.
  - `FORCE_PANIC_QUERY_ENGINE_GET_DMMF=1 npx prisma@3.15.0-integration-handle-fmt-panics.14 validate` 
  - An error prompt dialog should appear. Don't create any error report, but do select "Yes" when asked to create a new Github issue. The browser windows should open, the `prisma` process should get stuck.

2. The following block should open a new Github issue and terminate the `prisma` process.
  - `FORCE_PANIC_QUERY_ENGINE_GET_DMMF=1 npx prisma@3.15.0-integration-open-github-issues-on-windows-and-wsl.1 validate`
  - An error prompt dialog should appear. Don't create any error report, but do select "Yes" when asked to create a new Github issue. The browser windows should open, the `prisma` process should terminate.

### On other OS's

The following two blocks of steps should demonstrate the same behavior, i.e., terminating the `prisma` process after opening a new Github issue.

1. 
  - `FORCE_PANIC_QUERY_ENGINE_GET_DMMF=1 npx prisma@3.15.0-integration-handle-fmt-panics.14 validate`
  - An error prompt dialog should appear. Don't create any error report, but do select "Yes" when asked to create a new Github issue. The browser windows should open, the `prisma` process should terminate.

2.
  - `FORCE_PANIC_QUERY_ENGINE_GET_DMMF=1 npx prisma@3.15.0-integration-open-github-issues-on-windows-and-wsl.1 validate`
  - An error prompt dialog should appear. Don't create any error report, but do select "Yes" when asked to create a new Github issue. The browser windows should open, the `prisma` process should terminate.

### CODE REVIEW

File: packages/sdk/src/utils/getGithubIssueUrl.ts [hunk 1]
```diff
@@ -71,7 +73,15 @@ export async function wouldYouLikeToCreateANewIssue(options: IssueOptions) {
...
     })
-    await open(url)
+
+    /**
+     * This is a workaround for a quirky `open` behavior.
+     * - `await open(url)` correctly opens the browser, returns the control to Node.js and let it exit the process normally.
+     * - `await open(url, { wait: true })` achieves the same on Windows and WSL. Without `{ wait: true }`, `open` would not
+     *   return the control to the Node.js process, which would result in the prisma cli being stuck.
+     */
+    const shouldOpenWait = isWindows() || isWSL
```
Comment (janpio) [inline packages/sdk/src/utils/getGithubIssueUrl.ts:83]: Ugh, weird that one is a function and the other is not.

Comment (jkomyno) [reply packages/sdk/src/utils/getGithubIssueUrl.ts:83]: Yeah, `is-wsl` returns a boolean directly

**Comment by Weakky** [comment]
Hey,

Just tried on macOS on Apple silicon and it seemed to work as described

**Comment by janpio** [comment]
I tried this on Windows, and both versions successfully exited back to the terminal, but only in the 2. one I got the brower window as we want. (So different from your description, but still correct.)

**Review by aqrln — APPROVED** [review]

**Pull request merged**